packages/framework/src/Resources/config/services.yml

```diff
@@ -91,19 +92,21 @@ services:
         - '@Doctrine\Common\Cache\ArrayCache'
         - '@shopsys.shop.component.doctrine.cache.metadata.redis_cache'
 
+  Shopsys\FrameworkBundle\Model\AdminNavigation\BreadcrumbOverrider: ~
+
+  Shopsys\FrameworkBundle\Model\AdminNavigation\SideMenuBuilder:
+    public: true
```

Comment:
why is this public?

Reply:
The menu service must be public as it will be retrieved at runtime to keep it lazy-loaded.

https://symfony.com/doc/master/bundles/KnpMenuBundle/menu_builder_service.html